src/app/features/vendor/services/vendor.service.ts

```diff
@@ -0,0 +1,31 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
+import { IMarket } from '../models/vendor.model';
+import { IService } from '../models/vendor.model';
+import { IVendorCreation } from '../models/vendor.model';
```

Comment:
consolidate into a single import statement for line 4,5,6

Reply:
done